java-dynamic-memory-opts

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
I believe we need to add exec permission to this script, otherwise we will face `Permission denied` when the pod bootstraps.

Reply:
Added a `+x` flag to `java-dynamic-memory-opts` script.